src/containers/MapContext/reducer.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import { SET_LOCATION, SET_ADDRESS, SET_VALUES } from './constants';
+
+export const mapValuesType = PropTypes.shape({
+  location: PropTypes.shape({
+    lat: PropTypes.number.isRequired,
+    lng: PropTypes.number.isRequired,
+  }),
+  addressText: PropTypes.string,
+});
```

Comment:
This type should be included in `shared/types`

Reply:
reverted